extension/src/plots/model/collect.ts

```diff
@@ -818,9 +835,11 @@ export const collectOverrideRevisionDetails = (
   }
 
   return {
-    overrideOrder: comparisonOrder
-      .map(label => orderMapping[label])
-      .filter(Boolean),
+    overrideComparison: reorderObjectList(
```

Comment:
[F] Providing both sets of revisions here potentially in different orders means we don't have to worry about reordering later and simplifies the code outside of this collection process